Framework/TestHelpers/inc/MantidTestHelpers/FileComparisonHelper.h

```diff
@@ -1,6 +1,7 @@
 #ifndef FILECOMPARISONHELPER_H_
 #define FILECOMPARISONHELPER_H_
 
+#include <iterator>
```

Comment:
Yes, it uses `std::istreambuf_iterator` so it should have originally been included.

I'll also fix the merge conflict with release notes again